awx/api/urls.py

```diff
@@ -3,15 +3,26 @@
 
 # noqa
 
-from django.conf.urls import include, patterns, url as original_url
+import six
+from django.conf.urls import include, url as original_url
+from importlib import import_module
 
-def url(regex, view, kwargs=None, name=None, prefix=''):
-    # Set default name from view name (if a string).
-    if isinstance(view, basestring) and name is None:
-        name = view
-    return original_url(regex, view, kwargs, name, prefix)
 
-organization_urls = patterns('awx.api.views',
+def scoped_url(module):
```

Comment:
Since this `scoped_url` function is being used all over, should it live in a more "common" location, e.g. `awx/main/urls.py`. Could always throw it in `awx/main/utils` because we definitely need more random 💩 in there.